assets/prototype/application/ui/components/display/ErrorIndicator.js

```diff
@@ -1,14 +1,15 @@
+/**
```

Comment:
TS is very smart.
***
I was getting this error and did not understood at first what was the issue.
![image](https://user-images.githubusercontent.com/1477453/71646003-2ea83880-2ce9-11ea-8e7d-a5ec221776a0.png)

Reply:
Because of TS I found out, that these are returning boolean:
![image](https://user-images.githubusercontent.com/1477453/71646007-48498000-2ce9-11ea-9704-d39c2bb0ea47.png)

Reply:
So, error isn't an object with `code` and `message` as being destructured here: 
![image](https://user-images.githubusercontent.com/1477453/71646017-6f07b680-2ce9-11ea-9e6c-ce545a28e7d3.png)

Reply:
Also checked blueprint, title and description aren't required:
https://github.com/palantir/blueprint/blob/develop/packages/core/src/components/non-ideal-state/nonIdealState.tsx#L42

Reply:
oh I think originally that code may have been utilizing `ApolloError` objects, but ya TS is quickly becoming very loved

Reply:
@manzoorwanijk 's suggestion fixed this: https://github.com/eventespresso/event-espresso-core/pull/2161/commits/a891d00bcbba5240a44747e94e9ba54d0de17477